.github/workflows/deploy-wiki.yml

```diff
@@ -0,0 +1,37 @@
+name: Deploy wiki
+
+
+on:
+  push:
+    paths:
+      # Trigger only when wiki directory changes
+      - 'wiki/**'
+    branches:
+      # And only on master branch
+      - master
+
+jobs:
+  deploy-wiki:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with: 
+        # Fetch merge commit and last commit from pull request
+        fetch-depth: 2
+
+    - name: Set environment
+      run: |
+        # Set the author name, author email, and message from the pull request
+        echo "::set-env name=ACTION_MAIL::$(git log -n 1 --skip=1 --pretty=format:%ae)"
+        echo "::set-env name=ACTION_NAME::$(git log -n 1 --skip=1 --pretty=format:%an)"
+        echo "::set-env name=WIKI_PUSH_MESSAGE::$(git log -n 1 --skip=1 --pretty=format:%s)"
```

Comment:
I'm not in love with this method of extracting the original author. When a wiki pull request gets merged into master you get the following commit history:
```
commit c1a7a1cf0e61da8c8fc8227ffed21daec6e8e046 (HEAD -> master)
Merge: 160a357 e1c7ada
Author: maxb2 <maxb2@mail.missouri.edu>
Date:   Wed Jul 15 10:21:57 2020 -0500

    Merge branch 'throwaway700-patch-6'

commit e1c7adaa0302df6a7d41a908945eedee7cff1f6d (throwaway700-patch-6)
Author: throwaway700 <68336138+throwaway700@users.noreply.github.com>
Date:   Wed Jul 15 10:17:52 2020 -0500

    wiki update
````

So, you have to fetch the last two commits and skip one of them to get the author info. I'm open to other ways of doing this.

Reply:
Will this work if the commit gets squashed into a single commit?

Reply:
I don't think so. Can you think of another way to extract the author and commit message from a pull request? If we don't do this, we would still have the correct history in the original repo, but the wiki will always say dungeon-revealer is the author and have no commit message.

Reply:
I could have it check if the latest commit author is dungeon-revealer, skip back one commit and check again if true, else get the author and message.